manifests/a/ANTOrg-Inc/ASCIIEngine/1.0.0/ANTOrg-Inc.ASCIIEngine.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: ANTOrg-Inc.ASCIIEngine
+PackageVersion: 1.0.0
+PackageLocale: en-US
+Publisher: hdfsyu

```

```suggestion
Publisher: ANTOrg-Inc
```
This must match Control Panel.